server/routes/handlers/helpers/handleRecallFormPost.test.ts

```diff
@@ -4,16 +4,31 @@ import { handleRecallFormPost } from './handleRecallFormPost'
 import { updateRecall } from '../../../clients/manageRecallsApi/manageRecallsApiClient'
 import { validatePolice } from '../book/helpers/validatePolice'
 import { validateDecision } from '../assess/helpers/validateDecision'
+import * as referenceDataExports from '../../../referenceData'
 
 jest.mock('../../../clients/manageRecallsApi/manageRecallsApiClient')
 
 const handler = handleRecallFormPost(validatePolice, 'issues-needs')
 
+beforeEach(() => {
```

Comment:
seems strange that this is needed but the other refdata doesn't need to be mocked?

Reply:
It's needed now for the 2 tests in this class which use `/prison-police`.  I might move the mocking into just those specific tests.
```
const currentPageUrl = `/persons/${nomsNumber}/recalls/${recallId}/prison-police`
```

Reply:
Thinking to leave this as is for now also: I tried other ways to construct the mock response and tests were failing in all cases.

Reply:
👍